script.js

```diff
@@ -298,3 +307,38 @@ hamburgerButton.addEventListener('click', toggleMobileMenu);
 mobileMenuLinks.forEach((menuLink) => {
   menuLink.addEventListener('click', toggleMobileMenu);
 });
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (nameInput.value !== '' && messageInput.value !== '' && emailPattern.test(emailInput.value)) {
+    form.submit();
+    formErrorMessage.textContent = '';
+    emailErrorMessage.style.display = 'none';
+    nameErrorMessage.style.display = 'none';
+    messageErrorMessage.style.display = 'none';
+  } else {
+    formErrorMessage.textContent = 'the form was not sent';
+
+    if (!emailPattern.test(emailInput.value)) {
+      emailErrorMessage.textContent = 'the email input should follow this format : xyz@exe.com';
+      emailErrorMessage.style.display = 'block';
+    } else {
+      emailErrorMessage.textContent = '';
+      emailErrorMessage.style.display = 'none';
+    }
+    if (nameInput.value === '') {
+      nameErrorMessage.textContent = 'the name input is empty';
+      nameErrorMessage.style.display = 'block';
+    } else {
+      nameErrorMessage.textContent = '';
+      nameErrorMessage.style.display = 'none';
+    }
+
+    if (messageInput.value === '') {
+      messageErrorMessage.textContent = 'the message input is empty';
+      messageErrorMessage.style.display = 'block';
+    } else {
+      messageErrorMessage.textContent = '';
+      messageErrorMessage.style.display = 'none';
+    }
+  }
+});
```

Comment:
- [OPTIONAL] It would be nice to style the error message displayed on the mobile version of your application. 👍